app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,39 @@
+<head>
+  <script src="http://code.jquery.com/jquery-1.11.0.min.js"></script>
+  <%= javascript_pack_tag 'navbar', 'data-turbolinks-track': 'reload' %>
+</head>
+<body>
+  <nav class="navbar navbar-expand-lg navbar-light ">
+    <div class="container-fluid">
+      <div class="logo">
+        <%= image_tag "BCnameLogo.png", alt: "BCLogo", class: "logo-closed", id:"logo" %> 
+      </div>
+      <button id="button" class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarTogglerDemo02"
+        aria-controls="navbarTogglerDemo02" aria-expanded="false" aria-label="Toggle navigation" style="border: none; box-shadow: none">
+        <%= image_tag "list.svg", alt: "BCLogo", id:"image" %> 
+      </button>
+      <div class="collapse navbar-collapse " id="navbarTogglerDemo02">
+        <div class="navbar-nav ms-auto mb-2 mb-lg-0">
+          <li class="nav-item">
+            <a class="nav-link" aria-current="page" href="#" id="link">Nosotros</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" aria-current="page" href="#" id="link">Programas</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" aria-current="page" href="#" id="link">Graduados</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" aria-current="page" href="#" id="link">Convocatorias</a>
+          </li>
+          <li class="nav-item" id="borderContacto">
+            <a class="nav-link" aria-current="page" href="#" id="linkContacto">Contacto</a>
+          </li>
+          
+        </div>
+          <div class="group609">
+            <%= image_tag "Group 609.svg", alt: "group609" %> 
+          </div>
+      </div>
+  </nav>
+</body>
```

Comment:
Head and body tags are not necessary in this file, as they already exist in `application.erb.html`

```suggestion
  <nav class="navbar navbar-expand-lg navbar-light ">
    <div class="container-fluid">
      <div class="logo">
        <%= image_tag "BCnameLogo.png", alt: "BCLogo", class: "logo-closed", id:"logo" %> 
      </div>
      <button id="button" class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarTogglerDemo02"
        aria-controls="navbarTogglerDemo02" aria-expanded="false" aria-label="Toggle navigation" style="border: none; box-shadow: none">
        <%= image_tag "list.svg", alt: "BCLogo", id:"image" %> 
      </button>
      <div class="collapse navbar-collapse " id="navbarTogglerDemo02">
        <div class="navbar-nav ms-auto mb-2 mb-lg-0">
          <li class="nav-item">
            <a class="nav-link" aria-current="page" href="#" id="link">Nosotros</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" aria-current="page" href="#" id="link">Programas</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" aria-current="page" href="#" id="link">Graduados</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" aria-current="page" href="#" id="link">Convocatorias</a>
          </li>
          <li class="nav-item" id="borderContacto">
            <a class="nav-link" aria-current="page" href="#" id="linkContacto">Contacto</a>
          </li>
          
        </div>
          <div class="group609">
            <%= image_tag "Group 609.svg", alt: "group609" %> 
          </div>
      </div>
  </nav>

  <%= javascript_pack_tag 'navbar', 'data-turbolinks-track': 'reload' %>

```

Reply:
Ok! understood!

Reply:
@YilKanda ready! 👍